src/main/java/io/vlingo/http/Response.java

```diff
@@ -114,17 +118,22 @@ public int size() {
     return 9 + statusCode.length() + 1 + headersSize + 1 + entity.content().length() + 5;
   }
 
-  @Override
-  public String toString() {
-    final StringBuilder builder = new StringBuilder(size());
-
+  private <R> R into(Function<String,R> appender) {
     // TODO: currently supports only HTTP/1.1
+    appender.apply(Version.HTTP_1_1);
+    appender.apply(" ");
+    appender.apply(status.toString());
+    appender.apply("\n");
+
+    appendAllHeadersTo(appender);
+    appender.apply("\n");
+    return appender.apply(entity.toString());
+  }
 
-    builder.append(Version.HTTP_1_1).append(" ").append(status).append("\n");
-    appendAllHeadersTo(builder);
-    builder.append("\n").append(entity);
 
-    return builder.toString();
+  @Override
+  public String toString() {
+    return into(new StringBuilder(size())::append).toString();
```

Comment:
@VaughnVernon Any thoughts on this one? Should we pool the builder, or is it ok to create a new one each time since `into` now writes directly to the response buffer?

Reply:
@buritos I don't know without measuring both levels of overhead. Do you have an inclination based on `into()`?

Reply:
There is the `StringBuilder` overhead when using `toString`, but users can use `into` with a `ConsumerByteBuffer` they supply in order to avoid it.